src/common/drpc.c

```diff
@@ -45,7 +45,7 @@ daos_drpc_alloc(void *arg, size_t size)
 	struct drpc_alloc *alloc = arg;
 	void *buf;
 
-	D_ALLOC(buf, size);
+	DRPC_ALLOC(buf, size);
```

Comment:
Adding a DRPC_ALLOC() macro doesn't add anything.

"DRPC" allocs are fairly well mixed with "D" frees throughout the codebase, so you've never going to be able to track which allocator was used, so inherently they're going to have to use the same backend so there's no advantage in adding/changing this.